csv_detective/detect_fields/FR/geo/latitude_l93/__init__.py

```diff
@@ -1,18 +1,21 @@
 from frformat import LatitudeL93
 from csv_detective.detect_fields.other.float import _is as is_float
+
 from csv_detective.detect_fields.other.float import float_casting
 
 
 PROPORTION = 0.9
 
+_latitudel93 = LatitudeL93()
+
 
 def _is(val):
     try:
         if isinstance(val, (float, int)):
```

Comment:
Just seeing this now (I think it was introduced in https://github.com/datagouv/csv-detective/pull/87), but this condition could go, all tests are made on strings, so there should be no way to have ints/floats here